setup.py

```diff
@@ -1,21 +1,49 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
+
+__version__ = '0.1.0-dev'
+
+classes = """
+    Development Status :: 3 - ALpha
+    License :: OSI Approved :: BSD License
+    Environment :: Console
+    Framework :: IPython
+    Intended Audience :: Developers
+    Natural Language :: English
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Topic :: Software Development :: Libraries
+    Topic :: Text Processing :: Markup
+
 """
-ipymd
-"""
-from setuptools import setup
-
-setup(
-    name='ipymd',
-    version='0.1.0-dev',
-    packages=['ipymd',
-              'ipymd.core',
-              'ipymd.ext',
-              'ipymd.formats',
-              'ipymd.lib',
-              'ipymd.utils',
-              ],
-    entry_points={
-        'console_scripts': [
-            'ipymd=ipymd.core.scripts:main',
-            ]
-        }
-    )
+classifiers = [s.strip() for s in classes.split('\n') if s]
+
+description = ('Use the IPython notebook as an interactive Markdown editor')
+
+with open('README.md') as f:

```

Comment:
We're using the readme for the package's long description. This shows up on PyPI, and unfortunately only RST format is currently supported so your readme will likely not render correctly. You may want to update your readme to RST format in the future (kind of ironic, I know...).